oauth2_provider/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,197 @@
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+#: oauth2_provider/models.py:66
+msgid "Confidential"
+msgstr "Confidencial"
+
+#: oauth2_provider/models.py:67
+msgid "Public"
+msgstr "Público"
+
+#: oauth2_provider/models.py:76
+msgid "Authorization code"
+msgstr "Código de autorización"
+
+#: oauth2_provider/models.py:77
+msgid "Implicit"
+msgstr "Implícito"
+
+#: oauth2_provider/models.py:78
+msgid "Resource owner password-based"
+msgstr "Propiedario del recurso basado en contraseña"
+
+#: oauth2_provider/models.py:79
+msgid "Client credentials"
+msgstr "Credenciales de cliente"
+
+#: oauth2_provider/models.py:80
+msgid "OpenID connect hybrid"
+msgstr "OpenID connect híbrido"
+
+#: oauth2_provider/models.py:87
+msgid "No OIDC support"
+msgstr "Sin soporte para OIDC"
+
+#: oauth2_provider/models.py:88
+msgid "RSA with SHA-2 256"
+msgstr "RSA con SHA-2 256"
+
+#: oauth2_provider/models.py:89
+msgid "HMAC with SHA-2 256"
+msgstr "HMAC con SHA-2 256"
+
+#: oauth2_provider/models.py:104
+msgid "Allowed URIs list, space separated"
+msgstr "Lista de URIs permitidas, separadas por espacio"
+
+#: oauth2_provider/models.py:113
+msgid "Hashed on Save. Copy it now if this is a new secret."
+msgstr "Encriptadas al guardar. Copiar ahora si este es un nuevo secreto."
+
+#: oauth2_provider/models.py:175
+#, python-brace-format
+msgid "Unauthorized redirect scheme: {scheme}"
+msgstr "Esquema de redirección no autorizado"
+
+#: oauth2_provider/models.py:179
+#, python-brace-format
+msgid "redirect_uris cannot be empty with grant_type {grant_type}"
+msgstr "redirected_uris no pueden estar vacías para el tipo de autorización {grant_type}"
+
+#: oauth2_provider/models.py:185
+msgid "You must set OIDC_RSA_PRIVATE_KEY to use RSA algorithm"
+msgstr "Debes seleccionar OIDC_RSA_PRIVATE_KEY para usar el algoritmo RSA"
+
+#: oauth2_provider/models.py:194
+msgid "You cannot use HS256 with public grants or clients"
+msgstr "No es posible usar HS256 con autorizaciones o clientes públicos"
+
+#: oauth2_provider/oauth2_validators.py:211
+msgid "The access token is invalid."
+msgstr "El token de acceso es inválido."
+
+#: oauth2_provider/oauth2_validators.py:218
+msgid "The access token has expired."
+msgstr "El token de acceso ha expirado."
+
+#: oauth2_provider/oauth2_validators.py:225
+msgid "The access token is valid but does not have enough scope."
+msgstr "El token de acceso es válido pero no tiene suficiente alcance."
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:6
+msgid "Are you sure to delete the application"
+msgstr "¿Está seguro de eliminar la aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:12
+#: oauth2_provider/templates/oauth2_provider/authorize.html:29
+msgid "Cancel"
+msgstr "Cancelar"
```

Comment:
Seems like a command, so `Cancela`? i'm not hispanic, so just trying to recollect...

Reply:
As you can see here, in spanish our verbs ends on "ar", "er", or "ir". The correct translation for "he cancel" is "él cancela" but for "to cancel" te translation is "Cancelar". In the next image you can see that windows tools use the verbs ending in ar, er, and ir. 

![image](https://user-images.githubusercontent.com/30764904/170996669-b2323fc1-b0df-4c77-821f-4b6b94fc1b0c.png)

Reply:
sounds good. Thanks!